app/src/main/java/co/nimblehq/compose/crypto/ui/screens/detail/Appbar.kt

```diff
@@ -0,0 +1,74 @@
+package co.nimblehq.compose.crypto.ui.screens.detail
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Style
+import co.nimblehq.compose.crypto.ui.theme.Style.coinIfoAppBarIconColor
+import co.nimblehq.compose.crypto.ui.theme.Style.textColor
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun Appbar(
+    modifier: Modifier
+) {
+    BoxWithConstraints(modifier = modifier.fillMaxWidth()) {
+        IconButton(
+            modifier = Modifier.align(Alignment.CenterStart),
+            onClick = { /*TODO*/ }
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_back),
+                tint = MaterialTheme.colors.coinIfoAppBarIconColor,
+                contentDescription = null
+            )
+        }
+
+        Text(
+            modifier = Modifier.align(Alignment.Center),
+            // TODO: Remove dummy value when work on Integrate.
+            text = "Ethereum",
+            color = MaterialTheme.colors.textColor,
+            style = Style.medium16()
+        )
+
+        IconButton(
+            modifier = Modifier.align(Alignment.CenterEnd),
+            onClick = { /*TODO*/ }
+        ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_heart),
+                tint = MaterialTheme.colors.coinIfoAppBarIconColor,
+                contentDescription = null
+            )
+        }
+    }
+}
+
+@Suppress("FunctionNaming")
+@Preview
+@Composable
+fun AppbarPreview() {
+    ComposeTheme {
+        Surface {
+            Appbar(modifier = Modifier)
+        }
+    }
+}
+
+@Suppress("FunctionNaming")
+@Preview
```

Comment:
We can use this instead of darkTheme param

```suggestion
@Preview(uiMode = UI_MODE_NIGHT_YES)
```